listener.py

```diff
@@ -82,6 +85,11 @@ async def inspect_next_block(
 
         update_latest_block(inspect_db_session, block_number)
 
+        broker = connect_broker()
+        export_actor = dramatiq.actor(export_block_task, broker=broker)
+        logger.info(f"Sending block {block_number} for export")
```

Comment:
wrap in a function, but only create the broker / create the actor once

Reply:
otherwise we’re disconnecting and reconnecting to redis a bunch of times when it’s ok to just have the connection

Reply:
updated, let me know if that is what you mean